lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -19,12 +19,13 @@
           </p>
         </div>
         <div ng-if="main.selection.hasImage()">
-          <div class="step-selection-text">
+          <div class="step-selection-text"
+            uib-popover="{{ main.selection.getImagePath() }}"
+            popover-title="{{ main.selection.getImagePath() | basename }} - {{ main.selection.getImageSize() | gigabyte | number:1 }}GB"
+            popover-trigger="{'mouseenter': 'none', 'outsideClick': 'outsideClick'}">
```

Comment:
Since we might want all popovers across the app to behave the same way, I wonder if we can set the triggers globally using the `$uibTooltipProvider` provider (https://angular-ui.github.io/bootstrap/#!#popover). They seem to offer a `setTriggers(obj)` that we could use in `lib/gui/app.js` for example.